lib/web_console/middleware.rb

```diff
@@ -28,6 +28,12 @@ def call(env)
       request = create_regular_or_whiny_request(env)
       return @app.call(env) unless request.from_whitelited_ip?
 
+      if request.xhr?
+        unless request.acceptable?
```

Comment:
@gsamokovarov I found we need to accept normal requests (not XHRs). Which one do you like better, or any other way?

```ruby
  1
  # Returns whether the request is acceptable.
  def acceptable?
    (! xhr?) || accepts.any? { |mime| Mime::WEB_CONSOLE_V2 == mime }
  end
```

```ruby
  2
  # Returns whether the request is acceptable.
  def acceptable?
    (not xhr?) || accepts.any? { |mime| Mime::WEB_CONSOLE_V2 == mime }
  end
```